users/variables.tf

```diff
@@ -33,14 +33,14 @@ variable "assume_access_terraform_backend_policy_name" {
   default     = "Terraform-AssumeAccessTerraformBackend"
 }
 
-variable "assume_iam_admin_policy_description" {
-  description = "The description to associate with the IAM policy that allows assumption of the role to become an IAM admininistrator."
-  default     = "Allow assumption of the IamAdministrator role."
+variable "assume_provision_account_policy_description" {
```

Comment:
For consistency, this should be `assume_provisionaccount_policy_description`.  See also lines 41, 61, and 66.

Reply:
I did the same thing in the "shared-services" directory in #3!  I must remember incorrectly what we agreed to.